### DISCUSSION TITLE
Example of code for Distorting ops

### DISCUSSION THREAD

**Discussion by algorni** [dlemstra/Magick.NET#1345] [Help]
Hi, would be possible to have an example of code on how to achieve this:

https://imagemagick.org/Usage/distorts/#perspective

**Comment by algorni** [comment]
<img width="896" alt="image" src="https://user-images.githubusercontent.com/45007019/216969117-feb1919f-9ee8-4ee7-9e7f-cc740cda67a8.png">
To give more an idea, i could have picture of non-smart meter and i would like to reduce the distortion.

I have the 4 point of the red bounding box which may not be a rectangle according to the camera distortion.

**Reply by dlemstra** [reply]
What have you tried so far yourself and where are you stuck?
